### ISSUE TITLE
[E::bcf_translate] Unchecked error (2) at chr10:10331, exiting

### ISSUE THREAD

**Issue by hanguojun007** [samtools/bcftools#1741]
Hi，when I concatenate some vcf file，it report this error. I don't understand the `at chr10:10331` . What does that mean?

**Comment by hanguojun007** [comment]
Oh，this is the code: `bcftools concat -a *.gz -D -Oz -o all.vcf.gz`.

**Comment by pd3** [comment]
This is a generic error propagated from inside htslib and occurs when the program attempts to write a VCF/BCF line with unresolved tags. In plain English: the input VCF most likely contains tags that are not defined in the header. I think that's probably causing the problem. To verify, you can run
```
bcftools view input.vcf.gz -o /dev/null
``` 
and check for messages such as
```
[W::vcf_parse_info] INFO 'X' is not defined in the header, assuming Type=String
```
